CHANGELOG.md

```diff
@@ -29,18 +29,18 @@
    * FIXED: Update exit logic for non-motorways [#2252](https://github.com/valhalla/valhalla/pull/2252)
    * FIXED: Transition point map-matching. When match results are on a transition point, we search for the sibling nodes at that transition and snap it to the corresponding edges in the route. [#2258](https://github.com/valhalla/valhalla/pull/2258)
    * FIXED: Fixed verbal multi-cue logic [#2270](https://github.com/valhalla/valhalla/pull/2270)
-   * FIXED: Fixed Uturn cases when a not_thru edge is connected to the origin edge. [2272](https://github.com/valhalla/valhalla/pull/2272)
```

Comment:
other than the new entry at the bottom we fixed up where the `#` sign was missing